guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -7,73 +7,89 @@ You can register a host by using registration templates and set up various integ
 ifdef::katello,satellite,orcharhino[]
-. In the *Activation Keys* field, enter one or more activation keys to assign to hosts.
+. In the *Activation Keys* field, enter one or more activation keys to assign to your host.
 endif::[]
-. Optional: Select the *Insecure* option, if you want to make the first call insecure.
-During this first call, hosts download the CA file from {Project}.
-Hosts will use this CA file to connect to {Project} with all future calls making them secure.
+. Optional: If you want to use `wget` to register your host to {Project}, select the *Use wget* option.
+By default, {Project} generates a `curl` command.
+. Optional: If you want to make the first call insecure, select the *Insecure* option.
+During this first call, your host downloads the CA file from {Project}.
+Your host will use this CA file to connect to {Project} with all future calls making them secure.
 +
 {Team} recommends that you avoid insecure calls.
 +
-If an attacker, located in the network between {Project} and a host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from the registered host and the JSON Web Tokens (JWT).
-Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access the host using the SSH key.
+If an attacker, located in the network between {Project} and your host, fetches the CA file from the first insecure call, the attacker will be able to access the content of the API calls to and from your host and the JSON Web Tokens (JWT).
+Therefore, if you have chosen to deploy SSH keys during registration, the attacker will be able to access your host using the SSH key.
 +
-Instead, you can manually copy and install the CA file on each host before registering the host.
+Instead, you can manually copy and install the CA file on each host before registering.
 +
 To do this, find where {Project} stores the CA file by navigating to *Administer* > *Settings* > *Authentication* and locating the value of the *SSL CA file* setting.
 +
-Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on hosts and enter the following commands:
+Copy the CA file to the `/etc/pki/ca-trust/source/anchors/` directory on each host and enter the following commands:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # update-ca-trust enable
 # update-ca-trust
 ----
 +
-Then register the hosts with a secure `curl` command, such as:
+Then register your host with a secure `curl` command, such as:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # curl -sS https://{foreman-example-com}/register ...
 ----
 +
-The following is an example of the `curl` command with the `--insecure` option:
+The following is an example of the insecure `curl` command with the `--insecure` option:
 +
 [options="nowrap", subs="+quotes,attributes"]
 ----
 # curl -sS --insecure https://{foreman-example-com}/register ...
 ----
```

Comment:
```suggestion
The following is an example of the insecure `curl` command:
+
[options="nowrap", subs="+quotes,attributes"]
----
# curl -sS *--insecure* https://{foreman-example-com}/register ...
----
```

Reply:
I am unsure about highlighting parts of the command; this seems rather unusual. To reduce the overall length, maybe we can instead append a line such as 

````
+
If your host does not trust the {Project} SSL certificate, use the `X` option.
````

Reply:
Thinking about the whole "Insecure" bullet point IMO it would also be a good idea to omit the whole command examples. Users should simply copy the generated command and not modify it manually (as it is really a complex command) and the command details could make users think, they have to enter / modify things manually.

So we could just replace the part beginning from `Then register your host with a secure `curl` command, such as:` by `Afterwards you can continue registering your hosts without the *Insecure* option.`

What do you think about?

Reply:
Good point. I agree.

Reply:
I would not duplicate all command examples. I'd rather mention that hosts have to trust the ProjectServer SSL cert, or, alternatively, add the suggestion as above: https://github.com/theforeman/foreman-documentation/pull/2376/files#r1331083168